app/views/jobseekers/job_applications/_show.html.slim

```diff
@@ -1,43 +1,50 @@
-= job_application_review(job_application, allow_edit: local_assigns[:allow_edit]) do |r|
-  - render "jobseekers/job_applications/job_application_review_sections", r: r, jobseeker: current_jobseeker
-  - r.with_above do
-    = yield
+= yield
```

Comment:
this looks a very strange change - it seems to alter the with_above section for the current scenario. Was that the intention? Not sure about this stray 'yield'  statement on the first line. Maybe this code would be better inserted at a higher level rather than in this partial

Reply:
would it not be simpler just to include them in the template rather than yielding on line 1?